kotlin/school-timetabling/src/main/resources/META-INF/resources/app.js

```diff
@@ -5,8 +5,24 @@ let demoDataId = null;
 let scheduleId = null;
 let loadedSchedule = null;
 
+// Color Picker: Based on https://venngage.com/blog/color-blind-friendly-palette/
+const BG_COLORS = [ "#009E73","#0072B2", "#D55E00", "#000000", "#CC79A7", "#E69F00","#F0E442","#0072B2","#F6768E","#C10020","#A6BDD7","#803E75","#007D34"];
+const FG_COLORS = [ "#FFFFFF","#FFFFFF", "#FFFFFF", "#FFFFFF", "#FFFFFF", "#FFFFFF","#000000","#FFFFFF","#FFFFFF","#FFFFFF","#000000","#FFFFFF","#FFFFFF"];
```

Comment:
Some background are white? Looks out of place. (`LARGE` dataset.)

<img width="282" height="101" alt="image" src="https://github.com/user-attachments/assets/7f4e4c1f-39f4-4d5d-86d9-fc81a690f0da" />

Reply:
I'll make a task for the Swagger UI. 
I have extended the color list to 24 colors, if that is not enough then we are using colours for the wrong thing. :)